tests/unit/imports/Topic.test.js

```diff
@@ -175,17 +175,17 @@ describe('Topic', function() {
             topicDoc.infoItems.push({
                 isOpen: false
             });
-            topicDoc.infoItems[0].isOpen = true;
-            expect(Topic.hasOpenActionItem(topicDoc)).to.be.true;
+            topicDoc.infoItems[0].isOpen = false;
+            expect(Topic.hasOpenActionItem(topicDoc)).to.be.false;
         });
 
         it('returns true if the topic has a closed action item (object method call)', function() {
```

Comment:
Hi,
you see this line behind the "it"? This is the human readable form of what this test should check. While you made the 2 failing unit tests green, you did not quite the right thing to get them green. Because currently the unit tests do not check anymore what it was once intended for. Please leave the "it..." logic of the test as it is and try to modify the InfoItem inside the test a way that is satisfies the current changed product code.

Reply:
In other words: both of your changed unit tests do not keep their promise what they actually test. The solution is not to reverse this logic... isOpen=true=>false & exoect...true => ...false. Please revert your changes.
Read carefully what this tests wants to check. Read carefully the program logic of the "to be tested method" Topic.hasActionItems. Then decide how to change your test to get it green.
In case my writing is to confuse - ask on slack!